telebot/apihelper.py

```diff
@@ -697,11 +697,11 @@ def send_video(token, chat_id, data, duration=None, caption=None, reply_to_messa
     if thumb:
         if not util.is_string(thumb):
             if files:
-                files['thumb'] = thumb
+                files['thumbnail'] = thumb
```

Comment:
I propose to rename "thumb" parameter to "thumbnal" in all api_helper and async_helper functions for consistency. This will not affect anything, but will make all methods and names consistent.

(It can be done in the separate update because it will not affect behaviour)

Reply:
Will do it in separate update then